tests/test_cases/issue_2255/test_issue2255.py

```diff
@@ -9,11 +9,14 @@
 
 # GHDL is unable to access signals in generate loops (gh-2594)
 # Verilator doesn't support vpiGenScope or vpiGenScopeArray (gh-1884)
+# VCS is unable to access signals in generate loops
```

Comment:
We need an cocotb issue for that describing the issue in a way that a Synopsys AE can understand what's going on. I'll then file a Solvnet issue.

Reply:
https://github.com/cocotb/cocotb/issues/4328